contracts/src/teleporter/registry/TeleporterUpgradeable.sol

```diff
@@ -56,19 +73,48 @@ abstract contract TeleporterUpgradeable is Context, ITeleporterReceiver, Reentra
      */
     event TeleporterAddressUnpaused(address indexed teleporterAddress);
 
+    function _getTeleporterUpgradeableStorage()
+        private
+        pure
+        returns (TeleporterUpgradeableStorage storage $)
```

Comment:
I prefer to leave as is, mostly because it is the referenced example in ERC7201, and also throughout OZ contracts it is also the convention. In terms of `$` looking like a special symbol, can help point out that `$` is not like regular local variables, and is only used in reference to the storage variable.